src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -741,13 +741,13 @@ private async Task InitSourceRepoListAsync(UserSettings settings, CancellationTo
         private ValueTask PopulateSourceRepoListAsync(IReadOnlyCollection<PackageSourceMoniker> packageSourceMonikers, string optionalSelectSourceName, CancellationToken cancellationToken)
```

Comment:
Let's rename this one to be specific to collection name.

```suggestion
        private ValueTask PopulatePackageSourcesAsync(IReadOnlyCollection<PackageSourceMoniker> packageSourceMonikers, string optionalSelectSourceName, CancellationToken cancellationToken)
```

Reply:
Renamed.